suitability/test/test_maot/.project

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this and below are some kind of Python UI file and don't need to be in the repo (can be added to .gitignore .project , .pydevproject  )